apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/FilterRad.tsx

```diff
@@ -14,16 +14,27 @@ import {
   oppgavetypefilter,
   OppgavetypeFilterKeys,
   SAKSBEHANDLERFILTER,
-  SaksbehandlerFilterKeys,
   YTELSEFILTER,
   YtelseFilterKeys,
 } from '~components/nyoppgavebenk/Oppgavelistafiltre'
 import { useFeatureEnabledMedDefault } from '~shared/hooks/useFeatureToggle'
 import { FEATURE_TOGGLE_KAN_BRUKE_KLAGE } from '~components/person/OpprettKlage'
 import { FlexRow } from '~shared/styled'
+import { OppgaveDTO } from '~shared/api/oppgaver'
+import { UNSAFE_Combobox } from '@navikt/ds-react'
 
-export const FilterRad = (props: { hentOppgaver: () => void; filter: Filter; setFilter: (filter: Filter) => void }) => {
-  const { hentOppgaver, filter, setFilter } = props
+export const FilterRad = (props: {
+  hentOppgaver: () => void
+  filter: Filter
+  setFilter: (filter: Filter) => void
+  alleOppgaver: OppgaveDTO[]
+}) => {
+  const { hentOppgaver, filter, setFilter, alleOppgaver } = props
+
+  const saksbehandlere = new Set(
+    alleOppgaver.map((oppgave) => oppgave.saksbehandler).filter((s): s is Exclude<typeof s, null> => s !== null)
```

Comment:
Hvorfor gjøre det så komplisert? Gjør det veldig tungt å lese da man må først finne ut hvilken type saksbehandler er (`string | null`) og så ha i bakhodet hva resultatet fra `Exclude` blir. 

`.filter(s => !!s)` vil ha nøyaktig samme resultat.

Reply:
Helt til saksbehandleren `"false"` logger på systemet for første gang :D

Reply:
ah nvm det funker da og, ignorer meg!

Reply:
![Screenshot 2023-10-06 at 14 10 36](https://github.com/navikt/pensjon-etterlatte-saksbehandling/assets/82504565/8efdbc97-cc94-4c55-bc59-a0b8a1bf42fd)

Typescript er ikke enig med deg

Reply:
Viser seg å være et kjent problem i Typescript, så du må visst bruke type guards allikevel. Bra jobba Microsoft! 

https://github.com/microsoft/TypeScript/issues/7657